pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "svgdigitizer"
+version = "0.11.0"
+license = {text = "GPL-3.0-or-later"}
```

Comment:
> If you are using a standard, well-known license, it is not necessary to use this field. Instead, you should one of the [classifiers](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#classifiers) starting with License ::. (As a general rule, it is a good idea to use a standard, well-known license, both to avoid confusion and because some organizations avoid software whose license is unapproved.)

from https://packaging.python.org/en/latest/guides/writing-pyproject-toml/

Reply:
True. Thank you.